scripts/builtin/lmpredict.dml

```diff
@@ -0,0 +1,21 @@
+#-------------------------------------------------------------
+#
+# Copyright 2019 Graz University of Technology
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#-------------------------------------------------------------
+
+m_lmpredict = function(Matrix[Double] X, Matrix[Double] w) return (Matrix[Double] y) {
+  y = X %*% w
```

Comment:
Please extend this to allow for specify intercept as well - in which case you would need to append a column of ones and/or scale / shift.

Reply:
For now it's sufficient to support intercept 1 and include a TODO for intercept 2 as this is a longer discussion